plasmapy/classes/tests/test_Plasma.py

```diff
@@ -97,7 +97,7 @@ def test_Plasma3D_derived_vars():
     assert np.allclose(test_plasma.alfven_speed.value, 10.92548431)
 
 
-class Test_PlasmaBlob(object):
+class Test_PlasmaBlob():
```

Comment:
You don't even need the parentheses, but meh. :D

Reply:
We'll leave them in case we want to inherit from the 3D Plasma class :wink: